Rasa_Bot/tests/conftest.py

```diff
@@ -0,0 +1,26 @@
+import json
+from pathlib import Path
+
+import pytest
+from rasa.shared.core.domain import Domain
+from rasa_sdk import Tracker
+from rasa_sdk.executor import CollectingDispatcher
+from rasa_sdk.types import DomainDict
+
+here = Path(__file__).parent.resolve()
+
+with open(here / "./data/initial_tracker.json", "r", encoding='utf-8') as f:
```

Comment:
Can you add a docstring? Especially because this is quite rasa specific and important to understand when writing tests.